nemo/collections/asr/parts/jasper.py

```diff
@@ -179,21 +179,58 @@ def forward(self, x):
 
 
 class SqueezeExcite(nn.Module):
-    def __init__(self, channels, reduction_ratio):
+    def __init__(
+        self, channels, reduction_ratio, context_window: int = -1, interpolation_mode='nearest', activation=None
+    ):
+        """
+        Squeeze-and-Excitation sub-module.
+
+        Args:
+            channels: Input number of channels.
+            reduction_ratio: Reduction ratio for "squeeze" layer.
+            context_window: Integer number of timesteps that the context
+                should be computed over, using stride 1 average pooling.
+                If value < 1, then global context is computed.
+            interpolation_mode: Interpolation mode of timestep dimension.
```

Comment:
what are the possible values here?

Reply:
Actually, we dont expose this parameter for the user to override. The paper just uses `nearest`. I'll create another arg to allow to override this and update the docstring with possible values.

Reply:
Fixed, added the `se_interpolation_mode` flag as a possible argument to `jasper`